src/node_shell_read.js

```diff
@@ -5,10 +5,10 @@
  */
 
 requireNodeFS = function() {
-  // We always initialize both of these together, so we can use
-  // either one as the indicator for them not being initialized.
   if (!fs) {
```

Comment:
could we not change this to `if (!nodePath)`.. this would save a couple of bytes for everyone and I don't see any harm in re-importing fs once.. ?

Reply:
Done.